packages/tools/piw-utils-internal/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Is this the same license as `packages/tools/piw-native-utils-internal/LICENSE`? Is there benefit to duplicating it in two places? Could we included in once somewhere common?

Reply:
done